nilearn/conftest.py

```diff
@@ -484,13 +537,26 @@ def img_maps():
 def _img_labels():
```

Comment:
```suggestion
def _img_labels(n_regions=None):
```

Reply:
Another point is that all fixtures has a private twin. Why this is necessary? Could we think of a way to have only the public ones?